### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by tehseen01**
# Description

- added navbar under component
- added style in same folder

Fixes # (#2)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# preview

Screenshot 2022-12-16 125541
Screenshot 2022-12-16 125719
Screenshot 2022-12-16 125641

### CODE REVIEW

File: src/components/navbar/navbar.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        </div>
+      </nav>
+
+      <div
+        className={`${stl.mobileNav} ${navIcon ? stl.show : ''}`}
+        onClick={handelClick}
+      >
+        <span></span>
+        <span></span>
+        <span></span>
```
Comment (aftabrehan) [inline src/components/navbar/navbar.js]:
```
                      <span />
                      <span />
                      <span />
```

File: src/components/navbar/navbar.module.scss [hunk 2]
```diff
@@ -0,0 +1,125 @@
+@import '/src/styles/common.scss';
```
Comment (aftabrehan) [inline src/components/navbar/navbar.module.scss]: @import 'styles/common';

File: src/components/navbar/navbar.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  const [expendNav, setExpendNav] = useState(false)
+
+  const handelClick = () => {
+    setNavIcon(!navIcon)
+    setExpendNav(!expendNav)
+  }
+
+  return (
+    <header className={stl.header}>
+      <nav className={`${stl.navbar} ${expendNav ? stl.expend : ''}`}>
```
Comment (aftabrehan) [inline src/components/navbar/navbar.js]: Use `clsx` here, see the example in button component.

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Excellent PR @tehseen01 💪🚀 💯

Just needs minor fixes and it's good to go 🚀

**Review by mrxshinji — APPROVED** [review]
All good. Thanks

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @tehseen01 💪 🚀 💯

**Pull request merged**